src/krkn_lib/prometheus/krkn_prometheus.py

```diff
@@ -108,7 +108,7 @@ def process_query(
 
     def process_alert(
         self, alert: dict[str, str], start_time: datetime, end_time: datetime
-    ):
+    ) -> (Optional[int], Optional[str]):
```

Comment:
@chaitanyaenr @paigerube14 these changes  are not needed for the `critical_alerts`  task and have been done because a misunderstanding between me and Ravi , but since the code is well tested, I'd keep them, who knows that in the future we wouldn't collect also the alert, thoughts?